File: resources/examples/fizzbuzz/fizzbuzz.rb
puts "\aHowdy!\a"
(1..100).each do |i|
	modThree = i%3==0;
	modFour = i%4==0;

	if modThree and modFour
		puts "FizzBuzz";
	elsif modThree
		puts "Fizz"
	elsif modFour
		puts "Buzz"
	else
		puts i;
	end
end

File: resources/examples/fizzbuzz/fizzbuzz.js
function fizzbuzz(times) {
	var result = [];

	for (var i = 1; i < times; i++) {
		var modThree = i % 3 == 0;
		var modFour = i % 4 == 0;

		if (modThree && modFour) {
			result.push("FizzBuzz");
		} else if (modThree) {
			result.push("Fizz");
		} else if (modFour) {
			result.push("Buzz");
		} else {
			result.push(i);
		}
	}

	return result;
}